.github/workflows/cucumber-tests.yml

```diff
@@ -42,29 +37,37 @@ jobs:
       - name: Build Docker Images
         env:
           DOCKER_BUILDKIT: '1'
-          CSO_DEBUG_ENABLED: 'false'
         run: |
           docker-compose build --parallel efiling-demo efiling-frontend efiling-api
 
       - name: Standup Docker Pods
         env:
+          CSO_DEBUG_ENABLED: 'false'
           KEYCLOAK_CREDENTIALS_SECRET: ${{secrets.EFILING_DEMO_KEYCLOAK_CREDENTIALS_SECRET}}
           KEYCLOAK_REALM: ${{secrets.KEYCLOAK_REALM}}
           KEYCLOAK_URL: ${{secrets.KEYCLOAK_URL}}
           REACT_APP_KEYCLOAK_REALM: ${{secrets.EFILING_DEMO_KEYCLOAK_REALM}}
           REACT_APP_KEYCLOAK_URL: ${{secrets.EFILING_DEMO_KEYCLOAK_URL}}
           KEYCLOAK_AUTH_SERVER_URL: ${{secrets.EFILING_DEMO_KEYCLOAK_URL}}
-          CSO_DEBUG_ENABLED: 'false'
           KEYCLOAK_SSL_REQUIRED: 'none'
           KEYCLOAK_RESOURCE: 'efiling-api'
         run: |
-          docker-compose up -d redis sftp clamav efiling-demo efiling-frontend efiling-api &disown
-          until $(curl --output /dev/null --silent --head --fail http://localhost:8080/actuator/health); do
-            printf 'efiling-api not up'
-            sleep 5
-          done
-          printf 'up'
-          mvn verify -Pconfig -DBCEID_USERNAME=${{secrets.BCEID_USERNAME}} -DBCEID_PASSWORD=${{secrets.BCEID_PASSWORD}} -DEFILING_DEMO_KEYCLOAK_CREDENTIALS_SECRET=${{secrets.EFILING_DEMO_KEYCLOAK_CREDENTIALS_SECRET}} -DKEYCLOAK_URL=${{secrets.KEYCLOAK_URL}}/realms/${{secrets.KEYCLOAK_REALM}}/protocol/openid-connect/token -DBASE_URI=${{secrets.BASE_URI}} -DBROWSER=${{secrets.BROWSER}} -DENV=${{secrets.ENV}} -f tests/pom.xml
+          docker-compose up -d redis sftp clamav efiling-demo efiling-frontend efiling-api
+
+      - name: Running Integration Tests
+        env:
+          DOCKERIZE_VERSION: v0.6.1
+        run: |
+          # Maven requires chrome driver.
+          wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
+          sudo apt-get -y install ./google-chrome-stable_current_amd64.deb
+
+          # We need to test if efiling-api pod and service is running before we can proceed.  Using dockerize to proceed only after efiling-demo:8080 can be reached.
+          wget https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz
+          tar -xzvf dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz
+
+          ./dockerize -wait http://127.0.0.1:8080 -timeout 120s mvn verify -Pconfig -DBCEID_USERNAME=${{secrets.BCEID_USERNAME}} -DBCEID_PASSWORD=${{secrets.BCEID_PASSWORD}} -DEFILING_DEMO_KEYCLOAK_CREDENTIALS_SECRET=${{secrets.EFILING_DEMO_KEYCLOAK_CREDENTIALS_SECRET}} -DKEYCLOAK_URL=${{secrets.KEYCLOAK_URL}}/realms/${{secrets.KEYCLOAK_REALM}}/protocol/openid-connect/token -DBASE_URI=${{secrets.BASE_URI}} -DBROWSER=${{secrets.BROWSER}} -DENV=${{secrets.ENV}} -f tests/pom.xml
+          echo $?
 
```

Comment:
Although the curl method works, I wanted to incorporate dockerize as it will be more versatile for future docker-compose dependency checks eg: Database Connects or MQ Connects.

The curl loop also was causing issue due to an endloss loop.  If the api didn't run correctly then the DC up would never complete.